lib/percy/capybara/loaders/base_loader.rb

```diff
@@ -143,6 +142,21 @@ def _resources_from_dir(root_dir, base_url: '/')
           resources
         end
 
+        # A simplified version of Find.find that only returns files and follows symlinks.
+        def _find_files(*paths)
```

Comment:
Simplified this method in a followup commit: https://github.com/percy/percy-capybara/commit/37109a3d627ee1a94d2d8e546bd734d6bf88f0ce — it's crazy that Ruby stdlib doesn't have a clean way to do this natively, but here we are.